lib/Doctrine/DBAL/Driver/Connection.php

```diff
@@ -39,9 +39,40 @@ public function quote(string $input) : string;
     public function exec(string $statement) : int;
 
     /**
-     * Returns the ID of the last inserted row or sequence value.
+     * Returns the ID of the last inserted row.
+     *
+     * This method returns a string representing the value of the auto-increment field from the last row inserted into
```

Comment:
Same as for sequences, why does it have to be a string?

Reply:
[Same comment](https://github.com/doctrine/dbal/pull/3759#discussion_r352723604) as above.